### ISSUE TITLE
Theme the /node/%/releases project pages ("Project releases by project" view).

### ISSUE THREAD

**Issue by klonos** [backdrop-ops/backdropcms.org#386]
Example for Libraries project: https://backdropcms.org/node/163/releases

![image](https://cloud.githubusercontent.com/assets/2423362/25318336/9befb7c2-28cf-11e7-833f-c32c4b973a36.png)

- We shouldn't be repeating the project name
- We need the version number (what is currently the text of the download link) to be a link to the respective project version node on b.org
- The "Download" header could become a link to the binary itself.
- The "Release notes" header should be a link to the GitHub URL (not rendering the link itself as the link text)

**Comment by klonos** [comment]
Alrighty, I've changed the view:

- Output fields instead of content.
- Fields:
    - "Project release: Is security update": custom output as `security-update` when TRUE; nothing when FALSE; excluded from display; used as as row class (format -> settings -> row class).
    - "Project release: Version": no label; excluded from display; used as the link text for the "Content: Link" field.
    - "Content: Link": no label; rewritten as `[version]`
    - "Project release: Download link": no label; excluded from display; used as the link `href` for the "Download" link of the "Project release: Download size" field.
    - "Project release: Download size": no label; rewritten as `<a href="[download_link]">Download</a> ([download_size])`
    - "Project release: Release link": no label; rewritten as `Release notes`; output as link

@jenlampton this should be enough to get you going on the styling I think 😉 ...you can use https://backdropcms.org/node/449/releases which has a security update

![image](https://cloud.githubusercontent.com/assets/2423362/25319016/1f5cab04-28dc-11e7-97f4-fc3893941cf9.png)

- Not sure if we can add the node creation/update date as a release date. Sometimes we edit these manually. When we do that, the node publish date might not be the same as the actual release date.
- Should we output this as a table instead?

**Comment by klonos** [comment]
...I've added the post date and rewritten it as "Release date" (it's easier to remove fields than to add and configure them) and while doing that noticed that we have an "Project release: Update status" field available. I've added that and included it to the classes that each row gets.

**Comment by jenlampton** [comment]
It's a start: https://backdropcms.org/node/449/releases

**Comment by klonos** [comment]
Yes, it is 😄 Thanx @jenlampton 👍

**Comment by klonos** [comment]
I'm thinking that the way it is now, perhaps it makes better sense to have this page render as a table. This way we won't have the "version" and "Release date" text repeated in every row (they can be table headers instead).

**Comment by klonos** [comment]
...and the download button moved to the end (last column if we do make this a table).

**Comment by jenlampton** [comment]
Maybe we should kick this back to design before we spend any more time on
development ;)

**Comment by Dinornis** [comment]
@klonos you may want to redact the bottom of your first screenshot.

**Comment by klonos** [comment]
@Dinornis 👍  ...I'm usually very cautious with these sorts of things, but that was my email from the old domain I was using in Greece. My server has been pretty good in filtering spam so far. Thank you nevertheless. Fixed 😉 

Yes. Good enough for now I guess ...perhaps add some spacing between the fields so that they are not crumbed up (looks unprofessional) and leave it like that till we have a design in place.

**Comment by daggerhart** [comment]
Attempt at a stop-gap design to that can be worked towards until there is an official one.

![backdropcms.org releases page design](https://cloud.githubusercontent.com/assets/1205329/25776462/bafbe14c-328c-11e7-9b07-74a9fb048e41.png)

**Comment by jenlampton** [comment]
@daggerhart I like it!  Can we make this the official one? :)

**Comment by klonos** [comment]
Yes, 'tsgoooood 👍  Perhaps omit the "Version" label since it will be repeating itself in a long list of releases. But I'm not fussed 😄
